django/template/backends/django.py

```diff
@@ -62,6 +63,9 @@ def origin(self):
         return self.template.origin
 
     def render(self, context=None, request=None):
+        return mark_safe(''.join(self.stream(context, request)))
```

Comment:
I guess `mark_safe()` is needed because the empty string is considered "unsafe"? In that case, it might be easier to understand the intention with: `mark_safe('').join(...)`